### DISCUSSION TITLE
SocketException Windows

### DISCUSSION THREAD

**Discussion by S-ecki** [serverpod/serverpod#227] [Q&A]
Hello community, I wanted to try setting up Serverpod on my Windows machine. I know that it is not officially supported yet, but maybe someone else has solved the same problem already.

After executing `serverpod create` and creatig the Postgres tables, I wanted to start with `dart ./bin/main.dart`.
However, I am getting following Exception:

```
SERVERPOD version: 0.9.8 mode: development
2022-06-26 12:57:14.546260Z Internal server error. Failed to bind socket.
SocketException: Failed to create server socket (OS Error: Failed to start accept), address = ::, port = 8081
#0      _NativeSocket.bind (dart:io-patch/socket_patch.dart:996:7)
<asynchronous suspension>

Insights listening on port 8081
2022-06-26 12:57:14.565581Z Internal server error. Failed to bind socket.
SocketException: Failed to create server socket (OS Error: Failed to start accept), address = ::, port = 8080
#0      _NativeSocket.bind (dart:io-patch/socket_patch.dart:996:7)
<asynchronous suspension>

Server id 0 listening on port 8080
```

This does happen on other Ports as well.
Can someone help me out?

**Comment by vlidholt** [comment]
I'm not en expert on Windows, but it sounds like the server is prevented to starting by your firewall. It's possible you will need to grant access for Dart to open those ports somewhere.

**Comment by JimWuerch** [comment]
Maybe something is already running on port 8080 or 8081.  Before running anything try doing a `netstat -a -b` to see if anything is on TCP port 8080 or 8081.
